GLOSSARY.md

```diff
@@ -0,0 +1,22 @@
+# Cog Glossary
+
+## Bundle
```

Comment:
Bundle - a package containing one or more commands and the attributes (rules, permissions, possible options, documentation, templates, etc.) that belong to each command. A bundle can contain all the code for the command execution. https://github.com/operable/cog/wiki/Commands-and-Bundles#bundles